lib/videojs5-hlsjs-source-handler.js

```diff
@@ -52,6 +52,22 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
 
         function load(source) {
             _hls.loadSource(source.src);
+
+            if (!_hls.config.autoStartLoad) {
+                var checkStartLoad = function(e) {
+                    e.target.removeEventListener(e.type, checkStartLoad);
+                    if (typeof _hls.streamController.levels !== 'undefined') {
+                        _hls.startLoad(_hls.config.startPosition);
+                    } else {
+                        var startLoad = function() {
+                            _hls.off(Hls.Events.MANIFEST_PARSED, startLoad);
+                            _hls.startLoad(_hls.config.startPosition);
+                        };
+                        _hls.on(Hls.Events.MANIFEST_PARSED, startLoad);
+                    }
+                };
+                _video.addEventListener('play', checkStartLoad);
```

Comment:
it might be better listening to video.js tech events instead? idk if it can work, but in principle it would be better i guess (also we are listening to the native `waiting` event here somewhere). but this should also work very well of course.